### PR TITLE
chore: check if it is denoified

### PR DESCRIPTION
**Pull request by watany-dev**
### Author should do the followings, if applicable

- [ ] Add tests
- [ ] Run tests
- [ ] `yarn denoify` to generate files for Deno

fixed: https://github.com/honojs/hono/issues/1365

### CODE REVIEW

**Comment by watany-dev** [comment]
OK! Checked
https://github.com/honojs/hono/actions/runs/5965506637/job/16183092645

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,6 +19,11 @@ jobs:
       - run: npm run lint
       - run: npm run build
       - run: npm test
+      - run: |
```
Comment (yusukebe) [inline .github/workflows/ci.yml:22]: It's better to run `npm run denoify` before `npm test`.

**Comment by watany-dev** [comment]
Thank you. I've started by implementing the simpler solution for now. If we encounter any issues later on, we can revisit it.

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
Please wait. I have another idea.

**Comment by yusukebe** [comment]
@watany-dev 

Could you check this PR #1378 ?

**Comment by watany-dev** [comment]
@yusukebe 

Yes, as I wrote over there, I have no objection to this great idea!

**Comment by yusukebe** [comment]
Duplicated #1378

**Pull request closed without merging**